CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Sending email to reviewers. [#839](https://github.com/rokwire/rokwire-building-blocks-api/issues/839)
 - Self certification information to talent [#878](https://github.com/rokwire/rokwire-building-blocks-api/issues/878)
+- Version endpoint to contribution building block  [#897](https://github.com/rokwire/rokwire-building-blocks-api/issues/897)
```

Comment:
Please add the new changes to the bottom of the section. Thanks.

Reply:
What do you mean? I tried to add the changes based on the issue number

Reply:
Oh so the orders of the number is okay. Just put the newer thing to the bottom side. Got it. Changed